File: app/src/main/java/com/unitpricecalculator/util/AlertDialogs.java
package com.unitpricecalculator.util;

import android.content.DialogInterface;
import android.os.Build.VERSION;
import androidx.appcompat.app.AlertDialog;

public final class AlertDialogs {

  private AlertDialogs() {}

  public static void materialize(AlertDialog alertDialog) {
    if (VERSION.SDK_INT < 28) {
      return;
    }

    alertDialog.getButton(DialogInterface.BUTTON_POSITIVE).setAllCaps(false);
    alertDialog.getButton(DialogInterface.BUTTON_NEUTRAL).setAllCaps(false);
    alertDialog.getButton(DialogInterface.BUTTON_NEGATIVE).setAllCaps(false);
  }

}

File: app/src/main/java/com/unitpricecalculator/util/AlertDialogs.kt
@file:JvmName("AlertDialogs")

package com.unitpricecalculator.util

import android.content.DialogInterface
import android.os.Build.VERSION
import androidx.appcompat.app.AlertDialog

fun AlertDialog.materialize() {
  if (VERSION.SDK_INT < 28) {
    return
  }

  getButton(DialogInterface.BUTTON_POSITIVE)?.isAllCaps = false
  getButton(DialogInterface.BUTTON_NEUTRAL)?.isAllCaps = false
  getButton(DialogInterface.BUTTON_NEGATIVE)?.isAllCaps = false
}